pkg/minikube/preload/preload.go

```diff
@@ -67,6 +67,17 @@ func remoteTarballURL(k8sVersion string) string {
 	return fmt.Sprintf("https://storage.googleapis.com/%s/%s", constants.PreloadedVolumeTarballsBucket, tarballName(k8sVersion))
 }
 
+// TarballExists returns true if there is a preloaded tarball
+// that can be used
+func TarballExists(k8sVersion, containerRuntime string) bool {
+	if containerRuntime != "docker" {
+		return false
+	}
+	url := remoteTarballURL(k8sVersion)
+	_, err := http.Get(url)
```

Comment:
Probably better to use `_, err := http.Head(url)` to avoid a the full `GET` return from the server. WDYT?

Reply:
I agree ! with @josedonizetti less network footbprint the better.